packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -811,6 +806,31 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
     scrollController = widget.tableController.verticalScrollController!;
 
     pinnedScrollController = ScrollController();
+  }
+
+  @override
+  void didUpdateWidget(covariant _Table<T> oldWidget) {
+    super.didUpdateWidget(oldWidget);
+
+    final dataChanged = widget.tableController.tableData !=
```

Comment:
nit: rename to notifiersChanged since we are actually checking all the notifiers we listen to and not just table data